src/Microsoft.Extensions.Configuration.AzureAppConfiguration/ConfigurationSourceExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
+{
+    /// <summary>
+    /// Provides extension methods for configuration sources.
+    /// </summary>
+    public static class ConfigurationSourceExtensions
+    {
+        /// <summary>
+        /// Determines whether the specified configuration source is an Azure App Configuration source.
+        /// </summary>
+        /// <param name="source">The configuration source to check.</param>
+        /// <returns><c>true</c> if the specified source is an Azure App Configuration source; otherwise, <c>false</c>.</returns>
```

Comment:
![image](https://github.com/user-attachments/assets/f4777d94-3cff-41b5-8aec-16c91a247922)

We have inconsistent comment for boolean returns. Could you unify them in this PR?

Reply:
done! none of the options shows them highlighted in VS.